st2tests/st2tests/fixtures/traces/traces/trace_multiple_components.yaml

```diff
@@ -0,0 +1,25 @@
+---
+trace_id : test-trace-3
+action_executions :
+    - object_id: 'action_execution_1'
```

Comment:
Can you please use real examples? This is a constant problem in our code. Fixtures don't represent real models.